tensorboard/plugins/graph/tf_graph_common/render.ts

```diff
@@ -334,15 +334,33 @@ export class RenderGraphInfo {
    */
   getNearestVisibleAncestor(name: string): string {
     let path = getHierarchicalPath(name);
-    for (let i = 0; i < path.length; i++) {
-      let nodeName = path[i];
+    let i = 0;
+    let renderNode : RenderNodeInfo = null;
```

Comment:
nit: TypeScript, unlike ML, prefers the spacing `let renderNode: RenderNodeInfo = null;` (no space before colon).